contents/blog/posthog-changelog.md

```diff
@@ -26,6 +26,13 @@ Every Friday we update this page with the latest new features, fixes, and update
 
 Feature flags are a staple feature for PostHog, but we've added a bunch of new ways to add them to your product this week, depending on which libraries you're using. You can now choose to see a payload, bootstrap, or local evaluation version of the snippet, making it easier to add feature flags to your app and start testing changes.
 
+#### Experimental: DOM explorer mode
+![posthog recording dom explorer](../images/blog/array/dom-recording.png)
+
+Session replays usually capture images, rather than the DOM, but engineer Ben White thought it would be 'a cool thing' to change that. And, presto, we now have an explorable DOM mode.
```

Comment:
```suggestion
Our session replays look like video, but we really we capture the DOM and make that look like video. Engineer Ben White thought it would be 'a cool thing' to use the DOM for debugging. And, presto, we now have an explorable DOM mode.
```

Reply:
Technically correct. The best kind of correct.